pkg/pillar/scripts/device-steps.sh

```diff
@@ -164,7 +164,9 @@ access_usb() {
             if ! $BINDIR/tpmmgr saveTpmInfo $TPMINFOTEMPFILE; then
                 echo "$(date -Ins -u) saveTpmInfo failed" > $TPMINFOTEMPFILE
             fi
-            if tar cf /mnt/dump/diag1.tar /persist/status/ /var/run/ /persist/log "/persist/newlog" $TPMINFOTEMPFILE; then
+            NETDUMPDIR="/persist/netdump"
+            [ -d "$NETDUMPDIR" ] || NETDUMPDIR=""
+            if tar cf /mnt/dump/diag1.tar /persist/status/ /var/run/ /persist/log /persist/newlog "$NETDUMPDIR" "$TPMINFOTEMPFILE"; then
```

Comment:
Seems like an empty NETDUMPDIR will result in tar putting everything else in the file, but then it will do a non-zero exit. So I don't think this does what you want.

Reply:
Right, this is one of those case where shellcheck made me to break my original intentions and introduce a bug :)
Fixed